src/main/java/com/openlattice/chronicle/data/ChronicleCoreAppConfig.java

```diff
@@ -19,27 +19,33 @@
     private final UUID participatedInEntitySetId;
     private final UUID metadataEntitySetId;
     private final UUID partOfEntitySetId;
+    private       UUID messagesEntitySetId;
     private final UUID notificationEntitySetId;
     private       UUID participantEntitySetId;
+    private       UUID sentToEntitySetId;
     private final UUID studiesEntitySetId;
 
     @JsonCreator
     public ChronicleCoreAppConfig(
             @JsonProperty( SerializationConstants.ENTITY_SET_ID ) UUID hasEntitySetId,
             @JsonProperty( SerializationConstants.ENTITY_SET_ID ) Optional<UUID> participantEntitySetId,
             @JsonProperty( SerializationConstants.ENTITY_SET_ID ) UUID participatedInEntitySetId,
+            @JsonProperty( SerializationConstants.ENTITY_SET_ID ) Optional<UUID> messagesEntitySetId,
```

Comment:
No change required for this pull request. I think we should follow up with another pull request.

`@JsonProperty( SerializationConstants.ENTITY_SET_ID )` is the same value for each param, which should definitely not be the case. Fortunately, this object is only used internally so I don't think the `@JsonProperty` annotation ever comes into play.

I recommend removing all instances of `@JsonProperty` and `@JsonCreator` from this file.